eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -130,6 +130,50 @@ public function testCreateLocation()
         ];
     }
 
+    /**
+     * Test for the createLocation() method.
+     *
+     * @see \eZ\Publish\API\Repository\LocationService::createLocation()
```

Comment:
Nowadays we rather try to do:
```suggestion
     * @covers \eZ\Publish\API\Repository\LocationService::createLocation
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/3066/commits/c5536b0a3ec34bbdc63a99ecd01b0cbb74705187